avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -334,15 +351,17 @@ def mainDfa2Aimec(avaDir, comModule, cfg):
 
     # Setup input from com1DFA and save file paths to dictionary
     suffix = ['pfd', 'ppr', 'pfv']
-    pathDict = fU.getDFADataPaths(avaDir, pathDict, cfg, suffix, comModule)
+    cfgSetup = cfg['AIMECSETUP']
+    pathDict = fU.getDFADataPaths(avaDir, pathDict, cfgSetup, suffix, comModule)
 
-    # Extract mb info
-    if comModule == 'com1DFAOrig':
-        pathDict = extractCom1DFAMBInfo(avaDir, pathDict)
-    elif comModule == 'com1DFA':
-        pathDict = getMBInfo(avaDir, pathDict, comModule)
-    else:
-        pathDict = getMBInfo(avaDir, pathDict, comModule)
+    if cfg['FLAGS'].getboolean('flagMass'):
+        # Extract mb info
+        if comModule == 'com1DFAOrig':
+            pathDict = extractCom1DFAMBInfo(avaDir, pathDict)
+        elif comModule == 'com1DFA':
+            pathDict = getMBInfo(avaDir, pathDict, comModule)
+        else:
+            pathDict = getMBInfo(avaDir, pathDict, comModule)
```

Comment:
both elif and else are the same, drop the elif

Reply:
done